starter-code/app.js

```diff
@@ -0,0 +1,27 @@
+// Require Express
+const express = require('express');
+
+// Express server handling requests and responses
+const app = express();
+
+
+app.use(express.static('public'));
+
+// our first Route:
+app.get('/home', (request, response, next) => {
```

Comment:
usually for the index.html you don't have to do /home, you can leave it as just / and when you type in url localhost:2000 it will display your index.html